src/table/__tests__/header-cell.test.tsx

```diff
@@ -72,6 +72,7 @@ function TestComponent(props: Partial<TableHeaderCellProps<typeof testItem>>) {
       columnId="id"
       cellRef={() => {}}
       tableRole={tableRole}
+      variant="container"
```

Comment:
Why container?

Reply:
Any variant would do here. The property is used to assign a style which is not tested in by these tests. However, as the property is required some variant needs to be given.